databaseAnnotator/src/main/java/org/cbioportal/database/DatabaseAnnotator.java

```diff
@@ -50,38 +50,35 @@
 @SpringBootApplication
 public class DatabaseAnnotator {
 
-    private static Logger log = Logger.getLogger(DatabaseAnnotator.class);
+    private static Logger LOG = Logger.getLogger(DatabaseAnnotator.class);
 
-    private static Options getOptions(String[] args){
+    private static Options getOptions(String[] args) {
         Options gnuOptions = new Options();
         gnuOptions.addOption("h", "help", false, "shows this help document and quits.");
         gnuOptions.addOption("i", "isoform", true, "Isoform desired for annotation. mskcc or uniprot");
         gnuOptions.addOption("s", "studies", true, "Colon (:) delimited studies)");
         return gnuOptions;
     }
 
-    private static void help(Options gnuOptions, int exitStatus){
+    private static void help(Options gnuOptions, int exitStatus) {
         HelpFormatter helpFormatter = new HelpFormatter();
         helpFormatter.printHelp("Darwin Pipeline", gnuOptions);
         System.exit(exitStatus);
     }
 
+    private static void launchJob(String[] args, String isoform, String studies) throws Exception {
+        ConfigurableApplicationContext ctx = new SpringApplicationBuilder(DatabaseAnnotator.class).web(false).run(args);
```

Comment:
this is new, disabling web also for the database annotator (not sure if this even still works, the databaseannotator i mean)